docs/styleguide.md

```diff
@@ -2,15 +2,14 @@
 
 These are the guidelines for writing Electron documentation.
 
-## Titles
+## Headings
 
 * Each page must have a single `#`-level title at the top.
-* Chapters in the same page must have `##`-level titles.
-* Sub-chapters need to increase the number of `#` in the title according to
+* Chapters in the same page must have `##`-level headings.
+* Sub-chapters need to increase the number of `#` in the heading according to
   their nesting depth.
-* All words in the page's title must be capitalized, except for conjunctions
-  like "of" and "and" .
-* Only the first word of a chapter title must be capitalized.
+* The page's title must follow [APA title case][title-case]
+* All chapters must follow [APA sentence case][sentence-case]
```

Comment:
We should probably be consistent with periods at the end of bullet points.
```suggestion
* The page's title must follow [APA title case][title-case].
* All chapters must follow [APA sentence case][sentence-case].
```

There are other instances further down in the PR but I don't really have the time to annotate them all at the moment.